deploy/crds/nodemaintenance_cr.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha1
+apiVersion: nodemaintenance.kubevirt.io/v1alpha1
```

Comment:
Please notice that in order to go GA this should be at least v1beta1, maybe it's worth to address both in a single PR.